packages/lit-node-client-nodejs/src/lib/helpers/get-claims-list.ts

```diff
@@ -0,0 +1,30 @@
+import { ClaimsList, NodeShare } from '@lit-protocol/types';
+
+/**
+ * Retrieves a list of claims from the provided response data.
+ * @param responseData The response data containing the claims.
+ * @returns An array of claims.
+ */
+export const getClaimsList = (responseData: NodeShare[]): ClaimsList => {
+  const claimsList = responseData
+    .map((r) => {
+      const { claimData } = r;
+      if (claimData) {
+        for (const key of Object.keys(claimData)) {
+          for (const subkey of Object.keys(claimData[key])) {
+            if (typeof claimData[key][subkey] == 'string') {
+              claimData[key][subkey] = claimData[key][subkey].replaceAll(
+                '"',
+                ''
+              );
```

Comment:
the node's response models contain extra `'` characters in serialized strings. This method strips the serialization characters left over from the nodes

Reply:
yea we do - the returned signature has double quotes within the string, so like:

 ```
'"5bad778bd70ece43616bad778bd7ad778bd70ece43616c0"'
``` 

keep in mind that all functions introduced in this `helper` directory have always been in the `lit-node-client` code, but i extracted them out of the functions so we can test these parsers in isolation, and also a place to check the response data structure in case that we are receiving different format in the future.